tests/suites/test_suite_pkcs7.function

```diff
@@ -125,7 +125,8 @@ void pkcs7_verify(char *pkcs7_file,
     TEST_ASSERT(file != NULL);
 
     datalen = st.st_size;
-    ASSERT_ALLOC(data, datalen);
+    /* Add 1 so that data is non-NULL for zero length input */
+    ASSERT_ALLOC(data, datalen + 1);
```

Comment:
The downside of doing `+ 1` for all `datalen` is that we won't get the benefit of buffer overflow checking on sanitised builds/valgrind. 

Thoughts around `datalen ? datalen : 1`?